lib/netsuite/actions/update.rb

```diff
@@ -36,6 +36,12 @@ def request_body
           hash['platformMsgs:record']['@platformMsgs:externalId'] = updated_record.external_id
         end
 
+        inner = hash.dig("platformMsgs:record", :content!)
+        if inner.keys.grep(/(.*):nullFieldList?/).any?
+          null_field_key = inner.keys.grep(/(.*):nullFieldList?/)
+          hash["platformMsgs:record"][:content!]["platformCore:nullFieldList"] = hash["platformMsgs:record"][:content!].delete null_field_key[0]
+        end
```

Comment:
Could you explain what's going on here? Could you pull this into a separate PR?

Reply:
I needed to extract the `NullFieldList` stuff and this part makes sure the `updated_record.record_type` isn't used for the `nullFieldList`